tests/api_workflow/test_api_workflow_upload_dataset.py

```diff
@@ -144,7 +144,7 @@ def failing_upload_sample(*args, **kwargs):
 
     def test_upload_dataset_twice_with_overlap(self):
 
-        all_sample_names = [f'sample_{i}.jpg' for i in range(10)]
+        all_sample_names = [f'img_{i}.jpg' for i in range(10)]
```

Comment:
why did you change this?

Reply:
Because the name needs to fit those used elsewhere: 
https://github.com/lightly-ai/lightly/blob/36e1bf05c78f928b7a75b7e253e51c00e5c79240/tests/api_workflow/mocked_api_workflow_client.py#L157
https://github.com/lightly-ai/lightly/blob/4f3e7328539683fdd4990d463da0891aaf92cc4d/tests/api_workflow/mocked_api_workflow_client.py#L309

Reply:
I think I used different names on purpose to test the case where new data is added to an existing dataset

Reply:
I just saw that they do indeed not depend on the other image names, I changed them back to a unique name.